homeassistant/components/tensorflow/image_processing.py

```diff
@@ -84,26 +98,16 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     # append custom model path to sys.path
     sys.path.append(model_dir)
 
-    try:
-        # Verify that the TensorFlow Object Detection API is pre-installed
-        # pylint: disable=unused-import,unused-variable
-        os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
-        import tensorflow as tf  # noqa
-        from object_detection.utils import label_map_util  # noqa
-    except ImportError:
-        # pylint: disable=line-too-long
+    os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
+    if label_map_util is None:
         _LOGGER.error(
             "No TensorFlow Object Detection library found! Install or compile "
             "for your system following instructions here: "
             "https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/installation.md"
         )  # noqa
         return
 
-    try:
-        # Display warning that PIL will be used if no OpenCV is found.
```

Comment:
Restore this import.

Reply:
Which?  I import cv2 at the top and check for it on line 110 below and log an error if it isn't found.  Will that work?